aptos-move/aptos-gas/src/transaction.rs

```diff
@@ -73,6 +67,35 @@ crate::params::define_gas_parameters!(
             "gas_unit_scaling_factor",
             1000
         ],
+        // Gas Parameters for reading data from storage.
+        [load_data_base: InternalGas, "load_data.base", 1],
+        [
+            load_data_per_byte: InternalGasPerByte,
+            "load_data.per_byte",
+            1
+        ],
+        [load_data_failure: InternalGas, "load_data.failure", 1],
+        // Gas parameters for writing data to storage.
+        [
+            write_data_per_op: InternalGasPerArg,
+            "write_data.per_op",
+            100
+        ],
+        [
+            write_data_per_new_item: InternalGasPerArg,
+            "write_data.new_item",
+            1000
```

Comment:
💯 

(Thought we'd have separate per byte cost for new items, but this makes more sense.)

Reply:
Well we could have separate per byte costs for new and existing items in the future, if we think they are necessary.